core/src/main/java/com/linecorp/armeria/common/HttpResponse.java

```diff
@@ -390,6 +390,40 @@ static HttpResponse of(Publisher<? extends HttpObject> publisher) {
         }
     }
 
+    /**
+     * Creates a new HTTP response of the permanent redirect to specific location.
+     */
+    static HttpResponse ofRedirect(String location) {
+        requireNonNull(location, "location");
+        return of(ResponseHeaders.of(RedirectType.PERMANENT.toHttpStatus(), HttpHeaderNames.LOCATION, location));
```

Comment:
Would recommend either requiring the redirect type or at least making the default temporary. If someone accidentally uses this for permanent redirect without knowing it could create an unrecoverable situation.

Reply:
I fixed it to `Temporary` redirection when not requiring redirection type.
Thank you.